html/changelogs/shaker_top_cap.yml

```diff
@@ -0,0 +1,13 @@
+author: Vrow, KingOfThePing
+
+delete-after: True
+
+changes:
+  - imagedel: "Removed old shaker sprite."
+  - imageadd: "Added a new shaker that is more metallic, with additional detachable top and cap, as well as in-hands for them."
```

Comment:
While I recognize that alot of stuff does change with this PR, it might be best to condense some of these, like the spriting and sound into a general rscadd.

Reply:
A general ``rscadd`` wouldn't work since it's a lot of sound files being added, and a lot of sprites being added as well in a unique ``.dmi`` instead where the old shaker used to be (in ``drinks.dmi`` for some god forsaken reason), so having them documented like this is better in case people need to find when specific sounds/sprites were added via the changelog tags